### PR TITLE
Use the help text when it is specified in the rule

### PR DESCRIPTION
**Pull request by r4k**
Use the help text if it is provided in the description, otherwise use the generated help text.

### CODE REVIEW

File: src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java [hunk 1]
```diff
@@ -259,6 +259,9 @@ private String safeName(String s) {
     }
 
     private String help(MetricRule rule, String unit, String statistic) {
+        if (rule.help != null) {
+            return rule.help;
+        }
```
Comment (grobie) [inline src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java]: Indentation seems to be off by two spaces compared to the rest of the file.

**Comment by beorn7** [comment]
@brian-brazil

**Comment by brian-brazil** [comment]
This looks good, please address @grobie's comment and if you could remove beorn's email from the commit that'd be appreciated.

**Comment by r4k** [comment]
Fixing line spacing

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**